styles/bbc/catppuccin.user.css

```diff
@@ -0,0 +1,70 @@
+/* ==UserStyle==
+@name BBC Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/bbc
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/bbc
+@version 1.0.0
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/bbc/catppuccin.user.css
+@description Soothing pastel theme for BBC
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire*", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+@-moz-document domain('bbc.co.uk') {
```

Comment:
```suggestion
@-moz-document domain('bbc.co.uk'), domain('bbc.com') {
```

Reply:
I made a conscious decision to leave this out for now because I can't visit it without (I think) modifying the user agent and/or using a VPN.

Reply:
And I can't test it either lol since it redirects me to bbc.com.